app/templates/base.html

```diff
@@ -8,6 +8,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta content="" name="description">
   <meta content="width=device-width, initial-scale=1" name="viewport">
+  <title>{% block title %}My Survey{% endblock %}</title>
```

Comment:
Does this need to be `My Study` to match the text from the banner?

Reply:
Yes! My mistake.